alerter/Alerter.java

```diff
@@ -0,0 +1,64 @@
+class Alerter {
+    static int alertFailureCount = 0;
+
+    static final int THRESHOLD_VALUE = 200;
+
+    static final String ENV_TYPE = "DEVELOPMENT";
+
+    enum EnvType {
+        DEVELOPMENT, PRODUCTION
+    }
+
+    static void alertInCelsius(float fahrenheit) {
+        NetworkAlert networkAlert = getNetworkAlert();
+        float celsius = (fahrenheit - 32) * 5 / 9;
+        int returnCode = networkAlert.alert(celsius, THRESHOLD_VALUE);
+        if (returnCode != 200) {
+            alertFailureCount += 1;
+        }
+    }
+
+    private static NetworkAlert getNetworkAlert() {
+        NetworkAlert networkAlert = null;
+        if (EnvType.DEVELOPMENT.name().equals(ENV_TYPE)) {

```

Comment:
This is the place where interface abstraction should be used, which will eliminate the need of if/else